test/crossfilter.spec.js

```diff
@@ -871,14 +871,14 @@ describe("crossfilter", () => {
       it("can use a samplingRatio < 1 with filterQuery", () => {
         dimension.samplingRatio(0.5)
         expect(dimension.writeTopQuery(Infinity)).to.eq(
-          "SELECT users.id FROM users WHERE  MOD(users.rowid * 265445761, 4294967296) < 2147483648 ORDER BY users.id DESC"
+          "SELECT users.id FROM users WHERE  MOD(users.rowid * 2654435761, 4294967296) < 2147483648 ORDER BY users.id DESC"
```

Comment:
Hey @uyanga-gb I think these are hit testing sampling queries correct? If metis is generating why should cross filter also generate the similar queries?